### PR TITLE
add python-gobject

### PR DESCRIPTION
**Pull request by k-okada**
Please add the following dependency to the rosdep database.

## Package name:

python-gobject

## Purpose of using this:

This is almsot as same as https://github.com/ros/rosdistro/pull/19652, but we want to install `python-gobject` for debian/ubuntu.

https://pkgs.org/download/python-gobject

## Links to Distribution Packages

ubuntu: https://packages.ubuntu.com/bionic/python-gobject
debian: https://packages.debian.org/buster/python-gobject

fedra : https://apps.fedoraproject.org/packages/python2-gobject/contents/
gentoo: https://packages.gentoo.org/packages/dev-python/pygobject

# Checks
 - [x ] This package is expected to build on the submitted rosdistro

### CODE REVIEW

**Comment by sloretz** [comment]
CI reports a few issues. For a Python 2 only key, maybe this would work?

* nix: https://github.com/NixOS/nixpkgs/blob/9d452bdebedb1c47adaabd40e61a742f9f07b54c/pkgs/top-level/python-packages.nix#L7546
* opensuse: http://download.opensuse.org/distribution/leap/15.2/repo/oss/x86_64/python2-gobject-3.34.0-lp152.2.3.x86_64.rpm
* Ubuntu Focal: https://packages.ubuntu.com/focal/python-gobject

```yaml
python-gobject:
  nixos: [pythonPackages.pygobject2]
  opensuse: [python2-gobject]
  ubuntu:
    '*': null
    focal: [python-gobject]
```

**Comment by k-okada** [comment]
@sloretz thanks, fixed CI

File: rosdep/python.yaml [hunk 1]
```diff
@@ -2017,6 +2017,20 @@ python-gnupg:
   rhel:
     '7': [python2-gnupg]
   ubuntu: [python-gnupg]
+python-gobject:
+  arch: [python2-gobject]
+  debian:
+    '*': null
+    buster: [python-gobject]
+    stretch: [python-gobject]
+  fedora: [python3-gobject]
```
Comment (ivanpauno) [inline rosdep/python.yaml:2026]: isn't this the python3 package?
It looks like you want to add the python2 version here, right?

File: rosdep/python.yaml [hunk 2]
```diff
@@ -2017,6 +2017,20 @@ python-gnupg:
...
+  gentoo: [dev-python/pygobject]
+  nixos: [pythonPackages.pygobject3]
```
Comment (ivanpauno) [inline rosdep/python.yaml:2028]: same here

File: rosdep/python.yaml (hunk 1 above)
Comment (k-okada) [reply rosdep/python.yaml]: @ivanpauno yes, you're right. removed fedra key because it does not provide python2 binding and fix nixos .

**Review by ivanpauno — APPROVED** [review]
LGTM!

**Pull request merged**